File: javascript/1182.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var matr = [[],[],[],[],[],[],[],[],[],[],[],[]];
var valores = input.split('\n');
var coluna = parseFloat(valores.shift()), oper = String(valores.shift()).trim();
var sum = 0, count = 0;

for(let i = 0; i < 12; i++){
    for(let j = 0; j < 12; j++){
        matr[i].push(parseFloat(valores[count]));
        count++;
    }
}

for(let i = 0; i < 12; i++){
    sum += matr[i][coluna];
}
console.log(oper == "S" ? sum.toFixed(1) : (sum / 12).toFixed(1));

File: C/1182.c
#include <stdio.h>

int main(){
    int coluna,i,j;
    double sum=0,matriz;
    char tipo;
    scanf("%d\n%c",&coluna,&tipo);
    
    for(i=0;i<12;i++){
        for(j=0;j<12;j++){
            scanf("%lf",&matriz);
            if(coluna == j){
                sum+=matriz;
            }
        }
    }
    printf("%.1lf\n", tipo == 'S' ? sum : sum/12.0);
    
    return 0;
}